opennms-pris-plugins/opennms-pris-plugins-xls/pom.xml

```diff
@@ -21,12 +21,16 @@
 
     <dependencies>
         <!-- XLS -->
-        <dependency>
-            <groupId>net.sourceforge.jexcelapi</groupId>
-            <artifactId>jxl</artifactId>
-            <version>${jexcelApiVersion}</version>
-        </dependency>
-
+    <dependency>
+        <groupId>org.apache.poi</groupId>
+        <artifactId>poi</artifactId>
+        <version>3.17</version>
```

Comment:
Can we define a "apachePoiVersion" property and use it here instead? This seems to be the current behaviour, anyways.

Reply:
Besides that, the `JexcelApiVersion` from line 19 is not used anymore and can probably be removed in the code as well.

Reply:
yup thanks